cmd/auth-provider-gcp/apis/credentialprovider/install/BUILD

```diff
@@ -0,0 +1,29 @@
+load("@io_bazel_rules_go//go:def.bzl", "go_library")
+
+go_library(
+    name = "go_default_library",
+    srcs = ["install.go"],
+    importmap = "k8s.io/kubernetes/vendor/k8s.io/kubelet/pkg/apis/credentialprovider/install",
```

Comment:
Seems like this should read?

    importmap = "k8s.io/cloud-provider-gcp/cmd/auth-provider-gcp/apis/credentialprovider/install",

Reply:
Right, fixed